### DISCUSSION TITLE
Option to handle paths with IDs as a single path (e.g., /foo/:id)

### DISCUSSION THREAD

**Discussion by destroytoday** [plausible/analytics#2106] [Anything Else]
Hi! I’m kicking the tires on Plausible for my app and really enjoying it. The one thing I noticed is that it treats pages with ID-based slugs, like `/foo/:id` as unique individual page views, instead of a single page. This makes those page views pretty much unusable without any sort of grouping, but it would be incredible to be able to specify that those UUIDs are are actually a variable/param.

**Comment by metmarkosaric** [comment]
hi @destroytoday! perhaps you can try to group those pages like this: https://plausible.io/docs/custom-locations

**Comment by destroytoday** [comment]
@metmarkosaric This works! It’d be magical for Plausible to auto-detect UUIDs, but I’ll take what I can get :) Thanks for the quick solution!
